accountsplus/forms.py

```diff
@@ -1,14 +1,27 @@
 from __future__ import unicode_literals
 
+from contextlib import contextmanager
+
 import django.forms
 from django.conf import settings
 from django.apps import apps
-from django.contrib.auth.forms import AuthenticationForm
+from django.contrib.auth.forms import AuthenticationForm, PasswordResetForm
 from django.contrib.admin.forms import AdminAuthenticationForm
+from django.utils import translation
 
 from captcha.fields import ReCaptchaField
 
 
+@contextmanager
```

Comment:
@nasief you have this defined already in `context_manager.py`

Reply:
@waseem-omar Done!